project-api/project-api-test/src/test/java/io/ballerina/projects/test/TestBuildProject.java

```diff
@@ -351,6 +353,36 @@ public void testJBallerinaBackend() {
         Assert.assertEquals(platformLibraries.size(), 4);
     }
 
+    @Test(description = "tests langlib files are not included in the jar file collection for executing the test")
+    public void testLanglibFilesAreNotIncluded() {
```

Comment:
We can follow the pattern used in earlier tests and have the test name declared as something that goes `testBuildProjectWithNoLangLibFilesIncluded`

Reply:
Done it in 109da6bb831653d9925d85160a0853cab5a0fe77 commit